projects/storefrontlib/src/cms-components/product/product-tabs/product-details-tab/product-details-tab.component.html

```diff
@@ -1,3 +1,20 @@
 <ng-container *ngIf="product$ | async as product">
-  <div class="container" [innerHTML]="product?.description"></div>
+  <div class="container">
+    <div
+      *ngIf="!showMore; else fullContent"
+      [innerHTML]="product?.description.substring(0, 400)"
+    ></div>
+    <ng-template #fullContent
+      ><div [innerHTML]="product?.description"></div
+    ></ng-template>
+
+    <div *ngIf="product?.description.length > 400">
+      <a href="javascript:" *ngIf="showMore" (click)="toggle()">{{
```

Comment:
This is what we have in "product list". I think we can use similar buttons.
```
    <button
      *ngIf="state.maxVisible > state.topVisible"
      (click)="decreaseVisibleValues()"
      class="cx-action-link"
      cxFocus="moreorless"
    >
      {{ 'productList.showLess' | cxTranslate }}
    </button>

    <button
      *ngIf="state.maxVisible > 0 && state.maxVisible < facet.values.length"
      (click)="increaseVisibleValues()"
      class="cx-action-link"
      cxFocus="moreorless"
    >
      {{ 'productList.showMore' | cxTranslate }}
    </button>
```

Reply:
done